_layouts/post.html

```diff
@@ -25,6 +25,11 @@ <h1 class="title">{{page.title}}</h1>
           <div class="text-base margin-bottom-2">
             <a href="{{ site.baseurl }}/about/updates/releases">View all release notes</a>
           </div>
+        {% elsif page.category == "newsletters" %}
```

Comment:
typo here?

Reply:
elsif is correct - https://shopify.github.io/liquid/tags/control-flow/